modules/lead/product-base/production.tf

```diff
@@ -151,3 +93,28 @@ resource "kubernetes_role_binding" "default_production_rolebinding" {
     namespace   = module.production_namespace.name
   }
 }
+
+resource "kubernetes_role" "ci_production_role" {
+  provider = kubernetes.production
+  metadata {
+    name      = "ci-production-role"
+    namespace = module.production_namespace.name
+
+    labels = {
+      "app.kubernetes.io/name"       = "ci"
+      "app.kubernetes.io/instance"   = "ci"
+      "app.kubernetes.io/managed-by" = "Terraform"
+    }
+
+    annotations = {
+      description = "Permission required for Continous Integration tools to get pods in production namespace"
```

Comment:
I realize you didn't write this in this PR, but this isn't accurate anymore since this role allows CI to do pretty much everything. let's update this description